src/auth/RequireLogin.js

```diff
@@ -2,14 +2,14 @@ import React from 'react';
 import { connect } from 'react-redux';
 import Header from '../app/Header';
 import Loading from '../widgets/Loading';
-import Error404 from '../statics/Error404';
+import Error403 from '../statics/Error403';
```

Comment:
It's more common to use code 401 for unauthenticated user. 403 is more for admin pages.